lib/validator.rb

```diff
@@ -0,0 +1,96 @@
+#
+# Implements airbnb validator logic
+#
+module Validator
+  extend ActiveSupport::Concern
+
+  # Contains more key than expected
```

Comment:
A validator I created along the way. I will extract it into a single library in the future.